.infrastructure/helm-chart/todoapp/templates/pv.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{.Chart.Name}}-pv
+  namespace: {{.Values.todoapp.namespace}}
+spec:
+  storageClassName: standard
+  persistentVolumeReclaimPolicy: Delete
+  accessModes:
+    - ReadWriteMany
+  capacity:
+    storage: {{.Values.todoapp.pvcStorageRequest}}
```

Comment:
Ensure that the `todoapp.pvcStorageRequest` value is defined in your `values.yaml` file. If it's not defined, this could lead to issues when creating the PersistentVolume. Consider providing a default value or handling the case where it might be undefined.